src/bindings/html/dom.js

```diff
@@ -106,10 +116,12 @@ function camelCaseToDashed(string) {
 }
 
 function getElementTranslation(view, langs, elem) {
-  const l10n = getAttributes(elem);
+  const id = elem.getAttribute('data-l10n-id');
+  const args = JSON.parse(
+    escapeHtml(elem.getAttribute('data-l10n-args')));
```

Comment:
does it make sense here to `if (args)` so that we don't run JSON.parse when we don't have args?